src/modules/tools/endstops/Endstops.cpp

```diff
@@ -146,6 +147,10 @@ void Endstops::on_module_loaded()
 // Get config
 void Endstops::on_config_reload(void *argument)
 {
+    if (THEKERNEL->robot->actuators.size() > 3) {
+        THEKERNEL->streams->printf("Please implement trim for >3 actuators");
```

Comment:
Note this only gets printed out to the uart which no one will see unless debugging. This should do something more drastic. but really until there is a case for endstops for more than three motors, nothing should be changed here, and remain hard coded for 3.